language/functional_tests/src/genesis_accounts.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
I think this is a bit too heavy. It's pretty clear we only use it once and there's extra cost cloning from the lazy static. How about removing this file and moving the initialization into `config::global::Config::build`?

Reply:
I would prefer to keep it like this, but I don't have a super strong opinion on it. I agree that it's a bit heavyweight, but this way where you go to add a new genesis account is very clear, and easy to do. 

To expand on it a bit more; this also means that the exact accounts that we allow this "magic" on is made clear and separated from, and not lost in,  the logic of building the config. 

In terms of cloning, I can look at getting rid of that since this should have a 'static lifetime. 

What do you think?

Reply:
I get your point. Yeah putting this in its own module improves readability. I'm good with it.

How about we keep the `genesis_accounts` module but replace the lazy_static const with a `fn make_genesis_accounts() -> BTreeMap<...>`? It feels awkward to have a reference in the config, even if it's static.